app/electron/main.js

```diff
@@ -41,6 +42,13 @@ async function createWindow() {
     // so I don't have to write another 'if' statement
     protocol.registerBufferProtocol(Protocol.scheme, Protocol.requestHandler);
   }
+  const loadMainProcess = () => {
+    const files = glob.sync(path.join(__dirname, "mainEvents/**/*.js"));
+    /* eslint-disable global-require, import/no-dynamic-require */
```

Comment:
do not disable eslint, let there be warnings

Reply:
It doesn't let me commit the changes if there are warnings